src/index.js

```diff
@@ -1,4 +1,4 @@
-const { Microfleet, ActionTransport } = require('@microfleet/core')
+const { Microfleet } = require('@microfleet/core')
 const path = require('path')
 const merge = require('lodash/merge')
 const confidence = require('ms-conf')
```

Comment:
```suggestion
const conf = require('ms-conf')
```

тк у нас confidence внутри и он не используется напрямую, чтобы не было конфуза с модулем чужим